app/routes/application.js

```diff
@@ -1,12 +1,28 @@
 import Ember from 'ember';
 import ApplicationRouteMixin from 'simple-auth/mixins/application-route-mixin';
 
+const { inject } = Ember;
+const { service } = inject;
+
 export default Ember.Route.extend(ApplicationRouteMixin, {
-  flashMessages: Ember.inject.service(),
+  flashMessages: service(),
+
   actions: {
     sessionInvalidationSucceeded(){
       this.get('flashMessages').success('auth.confirmLogout');
       this.transitionTo('login');
+    },
+
+    error(error, transition) {
+      transition.abort();
+
+      this.get('flashMessages').alert('general.transitionErrorMessage');
+
+      // Manage route-related errors
```

Comment:
I added that for future reference. If anyone wants to handle error differently in the future, one could use `Ember.onerror(error);` to handle/log the route rejection errors and also render the error template (`error.hbs`) automatically by `return true;` at the end.